### PR TITLE
Fix up some potential leaks, by calling close()

### PR DESCRIPTION
**Pull request by brendandburns**
Apropos of https://github.com/kubernetes-client/java/pull/95

I audited the rest of our code...

### CODE REVIEW

File: examples/src/main/java/io/kubernetes/client/examples/AttachExample.java [hunk 1]
```diff
@@ -42,7 +42,7 @@ public static void main(String[] args) throws IOException, ApiException, Interru
         Configuration.setDefaultApiClient(client);
 
         Attach attach = new Attach();
-        final Attach.AttachResult result = attach.attach("default", "nginx-2371676037-czqx3", true);
+        final Attach.AttachResult result = attach.attach("default", "nginx-4217019353-k5sn9", true);
```
Comment (lwander) [inline examples/src/main/java/io/kubernetes/client/examples/AttachExample.java]: How useful is this pod name? Looks like it was generated by a deployment

Comment (brendandburns) [reply examples/src/main/java/io/kubernetes/client/examples/AttachExample.java]: Well, this is example code :)

I could put `"nginx-example-pod"`

I suppose I could dynamically query and try to attach to the first pod I find, but I'm not sure how perfect an example needs to be.  (I could also take the pod as a command line argument I suppose, but it seems like something for a different PR)

Comment (lwander) [reply examples/src/main/java/io/kubernetes/client/examples/AttachExample.java]: Fair enough :) maybe we can revisit if someone files an issue but you're right, let's not complicate it

**Comment by brendandburns** [comment]
@lwander any other comments? @mbohlool any thoughts?

If not, I can haz lgtmz?

**Comment by mbohlool** [comment]
Will look and merge today.

**Pull request merged**